tests/bluetooth/bsim_bt/bsim_test_l2cap_stress/src/common.h

```diff
@@ -36,7 +36,7 @@ extern enum bst_result_t bst_result;
 	}
 
 
-#define WAIT_SECONDS 120                         /* seconds */
```

Comment:
wow is it really that much slower, or did you add a fat margin?

Reply:
Well, yeah it is - no margin. 
The issue is that the way the connections get laid out they now 'collide' with only minimal slot reserved. Before the default (faulty) slot reservation left plenty of space in the slots to send/receive.